docs/changelog.rst

```diff
@@ -2,7 +2,8 @@
 Changelog
 =========
 
-* :feature:`1662` Users are now able to manually input ledger actions such as Income, Donation, Loss, Expense, Dividends Income
+* :feature:`1920` Rotki now supports your own Kusama node along the public ones.
```

Comment:
Can you change the wording here in a subsequent PR? "Rotki now supports addition of a custom Kusama endpoint"

Reply:
yep!